src/backend/base/langflow/components/embeddings/ollama.py

```
@@ -57,7 +57,9 @@ async def update_build_config(self, build_config: dict, field_value: Any, field_
         if field_name in {"base_url", "model_name"} and not await self.is_valid_ollama_url(field_value):
             # Check if any URL in the list is valid
             valid_url = ""
-            for url in URL_LIST:
+            if self.base_url:
+                check_urls = [self.base_url, *URL_LIST]
+            for url in check_urls:
```

Pylint (3.3.7)

[error] Possibly using variable 'check_urls' before assignment

(E0606)